tests/ci/auto_release.py

```diff
@@ -0,0 +1,97 @@
+import argparse
+import logging
+
+
+from get_robot_token import get_best_robot_token
+import github
+from typing import List, Any, TypeVar
+from github_helper import GitHub
+
+T = TypeVar("T")
+
+
+def parse_args():
+    parser = argparse.ArgumentParser("Create cherry-pick and backport PRs")
+    parser.add_argument("--token", help="github token, if not set, used from smm")
+    parser.add_argument(
+        "--repo", default="ClickHouse/ClickHouse", help="repo owner/name"
+    )
+    parser.add_argument("--dry-run", action="store_true", help="do not create anything")
+    parser.add_argument(
+        "--release-after-days",
+        default=3,
+        help="do automatic release on the latest green commit after the latest release if the newest release is older than the specified days",
+    )
+    # TODO(antaljanosbenjamin): handle this
+    parser.add_argument(
+        "--debug-helpers",
+        action="store_true",
+        help="add debug logging for git_helper and github_helper",
+    )
+
+    return parser.parse_args()
+
+
+def paginated_list_to_list(paginated_list: github.PaginatedList.PaginatedList[T]) -> List[T]:
+    return [item for item in paginated_list]
+
+READY_FOR_RELEASE_CHECK_NAME = "Ready for release"
+
+def main():
+    args = parse_args()
+    if args.debug_helpers:
+        logging.getLogger("github_helper").setLevel(logging.DEBUG)
+        logging.getLogger("git_helper").setLevel(logging.DEBUG)
+    token = args.token or get_best_robot_token()
+    gh = GitHub(token)
```

Comment:
I wanted to say, that `per_page=100` is good to have, but it's already in the wrapper =)